lib/ansible/plugins/callback/sumologic.py

```diff
@@ -89,6 +89,9 @@ def send_event(self, url, state, result, runtime):
         else:
             ansible_role = None
 
+        if 'args' in result._task_fields:
```

Comment:
Can you please add same thing for `splunk` as well ? Also, add a change log entry for this change [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments)?

Reply:
Done